.buildkite/pipeline.testing.yml

```diff
@@ -8,6 +8,11 @@ env:
 
 steps:
   - label: ":aws: E2E tests in AWS"
+    plugins:
+      - grapl-security/vault-login#v0.1.0
+      - grapl-security/vault-env#v0.1.0:
+          secrets:
+            - PULUMI_ACCESS_TOKEN
```

Comment:
You'll also need to add the following to the job:
``` yaml
agents:
  queue: "pulumi-staging"
```
The default agent queue, which this step is currently configured to use, does not have access to `PULUMI_ACCESS_TOKEN`.